detox/utils/report.js

```diff
@@ -0,0 +1,397 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/* eslint-disable no-console, camelcase, no-process-env */
+
+const axios = require('axios');
+const fse = require('fs-extra');
+const xml2js = require('xml2js');
+
+const {ARTIFACTS_DIR} = require('./constants');
+
+const MAX_FAILED_TITLES = 5;
+
+let incrementalDuration = 0;
```

Comment:
Maybe tracking of this value can be moved inside `getAllTests`?

Reply:
I'll move this variable inside getAllTests. This was copied from the original script.

Reply:
Done

Reply:
Thanks! I think in the webapp it's used in recursive function, but here looks like not.